### PR TITLE
docs(guides) Extracting Boilerplate updates to v4

### LINKED ISSUE #2204: Recommendations from chapter "Extracting Boilerplate" do not work for webpack 4
For webpack 3 it works, but after updating to version 4 I get the error `"Error: webpack.optimize.CommonsChunkPlugin has been removed, please use config.optimization.splitChunks instead"`. And I can't find in the documentation how it doing in version 4.

### PR DESCRIPTION
**Pull request by EugeneHlushko**
* update runtime chunk technique to v4

Fixes #2204

### CODE REVIEW

File: src/content/guides/caching.md [hunk 1]
```diff
@@ -116,13 +116,12 @@ W> Output may differ depending on your current webpack version. Newer versions m
 
 ## Extracting Boilerplate
 
-As we learned in [code splitting](/guides/code-splitting), the [`CommonsChunkPlugin`](/plugins/commons-chunk-plugin) can be used to split modules out into separate bundles. A lesser-known feature of the `CommonsChunkPlugin` is extracting webpack's boilerplate and manifest which can change with every build. By specifying a name not mentioned in the `entry` configuration, the plugin will automatically extract what we want into a separate bundle:
+As we learned in [code splitting](/guides/code-splitting), the [`SplitChunksPlugin`](/plugins/split-chunks-plugin/) can be used to split modules out into separate bundles. webpack provides an optimization feature which does split out runtime code into a separate chunk(s) according to the options provided, simply use [`optimization.runtimeChunk`](/configuration/optimization/#optimization-runtimechunk) set to `single` for creating one runtime bundle:
```
Comment (EugeneHlushko) [inline src/content/guides/caching.md]: I am tempted to also remove
```
As we learned in [code splitting](/guides/code-splitting), the [`SplitChunksPlugin`](/plugins/split-chunks-plugin/) can be used to split modules out into separate bundles.
```

Any thoughts?

Comment (montogeek) [reply src/content/guides/caching.md]: No, it is always good to link more documentation.

Comment (EugeneHlushko) [reply src/content/guides/caching.md]: OK in that case suppose its good to go ?

Comment (montogeek) [reply src/content/guides/caching.md]: Yes

Comment (EugeneHlushko) [reply src/content/guides/caching.md]: Ok updating branch

**Comment by montogeek** [comment]
Thanks1

**Pull request merged**